testing_and_setup/compass/ocean/hurricane/forward_template.xml

```diff
@@ -15,8 +15,8 @@
 		<option name="config_ALE_thickness_proportionality">'weights_only'</option>
 		<option name="config_vert_coord_movement">'uniform_stretching'</option>
 
-                <option name="config_use_const_visc">.true.</option>
-                <option name="config_vert_visc">1.0e4</option>
+                <option name="config_use_cvmix_background">.true.</option>
+                <option name="config_cvmix_background_viscosity">1.0e4</option>
```

Comment:
same question here, seems like a really big number

Reply:
perhaps these two are supposed to be 1e-4?

Reply:
Pseudo-2D approach...